### PR TITLE
Fixing broken images on Knowledge Hub Noteboks Readme

### PR DESCRIPTION
**Pull request by benji-glitsos-ga**
### Proposed changes

Affected page: https://docs.dea.ga.gov.au/notebooks/README/

Using absolute github links for readme images to fix broken images on KH.

### CODE REVIEW

**Review by BexDunn — APPROVED** [review]
Nice one

**Comment by robbibt** [comment]
Hey @benji-glitsos-ga, happy with this if you think absolute links are necessary - do you know though why other pages like this work fine with relative links in Knowledge Hub?
https://github.com/GeoscienceAustralia/dea-notebooks/blob/f79e18063d8937dae4a76232cd659baa18251c77/Beginners_guide/02_DEA.ipynb#L169
https://docs.dea.ga.gov.au/notebooks/Beginners_guide/02_DEA.html#Surface-reflectance

**Comment by robbibt** [comment]
And should we point to stable, not develop?

**Comment by benji-glitsos-ga** [comment]
@robbibt The reason this is necessary is a finicky difference with how the Knowledge Hub handles the Readme's URL meaning that a relative path points to a slightly different location on the Readme than it does on the Sandbox. This is what caused the relative paths on the Knowledge Hub's Readme to break.

Absolute links are not ideal, but it is only temporary until I eventually replace this Knowledge Hub Notebooks page with a proper cover page.

This PR will need to be merged into stable. Im not sure -- it may be better to merge into develop first then merge into stable?

**Comment by robbibt** [comment]
> @robbibt The reason this is necessary is a finicky difference with how the Knowledge Hub handles the Readme's URL meaning that a relative path points to a slightly different location on the Readme than it does on the Sandbox. This is what caused the relative paths on the Knowledge Hub's Readme to break.
> 
> Absolute links are not ideal, but it is only temporary until I eventually replace this Knowledge Hub Notebooks page with a proper cover page.
> 
> This PR will need to be merged into stable. Im not sure -- it may be better to merge into develop first then merge into stable?

Ah, makes sense!

And re my comment above, I just meant use an absolute URL pointing to the `stable` branch, not `develop` like it currently is:
https://github.com/GeoscienceAustralia/dea-notebooks/pull/1197/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R1

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<img src="Supplementary_data/dea_logo_wide.jpg" width="900" alt="Digital Earth Australia logo" />
+<img src="https://raw.githubusercontent.com/GeoscienceAustralia/dea-notebooks/develop/Supplementary_data/dea_logo_wide.jpg" width="900" alt="Digital Earth Australia logo" />
```
Comment (robbibt) [inline README.md]:
```suggestion
<img src="https://raw.githubusercontent.com/GeoscienceAustralia/dea-notebooks/stable/Supplementary_data/dea_logo_wide.jpg" width="900" alt="Digital Earth Australia logo" />
```

File: README.md [hunk 2]
```diff
@@ -75,7 +75,7 @@ The **DEA Knowledge Hub** is where you can easily view the notebooks and read DE
 
 The **DEA Sandbox** is a free virtual environment for interacting with the notebooks and tools from your web browser. You can use the Sandbox to conduct geospatial analysis without installing anything on your computer. Learn how to [use the DEA Sandbox](https://docs.dea.ga.gov.au/guides/setup/Sandbox/sandbox/).
 
-<a href="https://docs.dea.ga.gov.au/guides/setup/Sandbox/sandbox/"><img src="Supplementary_data/DEA_Sandbox/DEA_Sandbox_cover_page.png" width="500" alt="The cover page of the DEA Sandbox." /></a>
+<a href="https://docs.dea.ga.gov.au/guides/setup/Sandbox/sandbox/"><img src="https://raw.githubusercontent.com/GeoscienceAustralia/dea-notebooks/develop/Supplementary_data/DEA_Sandbox/DEA_Sandbox_cover_page.png" width="500" alt="The cover page of the DEA Sandbox." /></a>
```
Comment (robbibt) [inline README.md]:
```suggestion
<a href="https://docs.dea.ga.gov.au/guides/setup/Sandbox/sandbox/"><img src="https://raw.githubusercontent.com/GeoscienceAustralia/dea-notebooks/stable/Supplementary_data/DEA_Sandbox/DEA_Sandbox_cover_page.png" width="500" alt="The cover page of the DEA Sandbox." /></a>
```

**Review by robbibt — APPROVED** [review]

**Pull request merged**